src/util-threshold-config.c

```diff
@@ -908,7 +908,7 @@ static int ParseThresholdRule(DetectEngineCtx *de_ctx, char *rawstr,
            break;
         case THRESHOLD_TYPE_SUPPRESS:
             /* need to get IP if extension is provided */
-            if (th_track != NULL) {
+            if (strcmp("", th_track) != 0) {
```

Comment:
I suggest that this check be removed. Before, it was necessary since `th_track` could be `NULL`.

Reply:
I think this check is still required as track is an optional parameter in this rule, so it may not be set?

Reply:
Ok.